tests/typescript/class/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -21,6 +21,10 @@ class A {
     }
 }
 
+class B {
+  constructor<>() {}
```

Comment:
Will it work if there are comments inside?

Reply:
No, class constructor in TypeScript does not accept type parameterss.